### DISCUSSION TITLE
Do not reference projects from xmake internal

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#3004] [Ideas]
![image](https://user-images.githubusercontent.com/42310255/198929626-f2fdde9f-2150-4c74-9edf-91267f792bd5.png)
如图，在检测包的reference的时候把很多xmake内部的地址都引用进去了。能不能在reference里面排除掉以xmake_globaldir开头的所有目录，或者在xmake clean的时候精简一下？

**Comment by waruqi** [comment]
这个放着也不影响什么吧，而且正好还能看出来什么时候哪些包 是通过 xrepo install 安装的。
